client/my-sites/upgrades/plans-compare/index.jsx

```diff
@@ -0,0 +1,42 @@
+/**
+ * External dependencies
+ */
+import { connect } from 'react-redux';
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import { getPlans } from 'state/plans/selectors';
+import { getPlansBySite } from 'state/sites/plans/selectors';
+import PlansCompare from 'components/plans/plans-compare' ;
+
+const UpgradePlansCompare = ( { domain, sites, plans, features, productsList } ) => {
+	const selectedSite = sites.getSelectedSite();
+	const backUrl = '/domains/add/' + domain + '/plans/' + selectedSite.slug;
+
+	return (
+		<div className="plans has-sidebar">
+			<PlansCompare { ...{ selectedSite, plans, features, productsList, backUrl } }
+				hideFreePlan={ true }
+				isInSignup={ true } />
```

Comment:
just a personal styling preference, but when I have these mixes, I tend to link to keep them one-per-line…

```js
<PlansCompare
	{ ...{
		backUrl
		features,
		plans,
		productsList,
		selectedSite
	} }
	hideFreePlan={ true }
	isInSignup={ true }
/>	
```

of course, you can merge the others into the spread object

```js
<PlansCompare { ...{
	hideFreePlan: true,
	isInSignup: true,
	backUrl
	features,
	plans,
	productsList,
	selectedSite
} } />	
```